modules/javafx.graphics/src/main/java/javafx/scene/TraversalDirection.java

```diff
@@ -22,20 +22,24 @@
  * or visit www.oracle.com if you need additional information or have any
  * questions.
  */
-
-package com.sun.javafx.scene.traversal;
+package javafx.scene;
 
 /**
- * Specifies the traversal method.
+ * Specifies the direction of focus traversal.
+ *
+ * @since 24
```

Comment:
I would add `@see` for the method that uses this enum, otherwise it's rather disconnected from anything.